authui/src/authgear.ts

```diff
@@ -8,6 +8,15 @@ window.api.onLoad(() => {
   document.body.classList.add("js");
 });
 
+// Handle history tracking.
+
+let inHistorySettings = false;
+window.api.onLoad(() => {
+  if (window.location.pathname === "/settings") {
```

Comment:
I just realized `/change_password` is not under `/settings`.

Reply:
I suppose we can move it under settings, so it'll be `/settings/change_password` and `/settings/change_secondary_password`.

I just remembered `/enter_login_id`, its name looks like part of interaction flow, yet it's part of settings page. Should we change its name too?

Reply:
> I suppose we can move it under settings, so it'll be /settings/change_password and /settings/change_secondary_password.

Agree.

> I just remembered /enter_login_id, its name looks like part of interaction flow, yet it's part of settings page. Should we change its name too?

Agree. Let's change from `/enter_login_id` to `/settings/enter_login_id`?